packages/app/src/App.tsx

```
@@ -0,0 +1,26 @@
+
+const App = () => {
+  return (
+    <>
+      <Routes>
+        <Route path="/nb/:notebookId" element={<MainPage />} />
+        <Route path="*" element={<NotFound />} />
+      </Routes>
+      {location.hash.startsWith("#/notebook") && (
+        <div className="fixed bottom-0 w-screen bg-border shadow-lg text-color px-4 py-1 flex justify-between">
+          <div>Citronote 3 is currently in the trial release phase.</div>
+          <a className="opacity-80">
```

Biome (2.1.2)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)